text/language_extensions.tex

```diff
@@ -187,6 +188,7 @@ \section{Interrupt state control}
 For now, assume that the thread is attempting to sleep for the number of cycles shown by the \c{printf} call at the start, outside of the loop.
 
 \codelisting[filename=examples/interrupts_disabled/interrupts.cc,marker=low,label=lst:lowpriorityinterruptsdisabled,caption="A low-priority thread that uses an interrupts-disabled function to consume CPU."]{}
+\amanda{The text above the code explaining interrupt_state(disabled) is "cheri::" but the text in the code is "cheriot::"  Should one of these be changed?}
```

Comment:
cheriot:: is the correct one.  We changed it and I may have missed some places in the text.